web/extensions/ant_switch/ant_switch.js

```diff
@@ -378,8 +398,8 @@ var ant_sw = {
              w3_disable(el, lock);
           }
  
-          // Ground All
-          var re=/^Ground all$/i;
+          // Switch All Off
```

Comment:
Ground All is same as Switch All Off. please scan the code to see if there is duplicated or dead logic around ground all. use them for switch all off.